diff --git a/terraform.tfvars b/terraform.tfvars
index 7e97861..1e642cd 100644
--- a/terraform.tfvars
+++ b/terraform.tfvars
@@ -95,11 +95,12 @@ k8s_coprocessor_deps = {
   default_namespace = "coproc"

   namespaces = {
-    coproc         = { enabled = true }
-    coproc-admin   = { enabled = true }
-    gw-blockchain  = { enabled = true }
-    eth-blockchain = { enabled = true }
-    monitoring     = { enabled = true }
+    coproc             = { enabled = true }
+    coproc-admin       = { enabled = true }
+    gw-blockchain      = { enabled = true }
+    eth-blockchain     = { enabled = true }
+    polygon-blockchain = { enabled = true }
+    monitoring         = { enabled = true }
   }

   service_accounts = {
